src/command_modules/azure-cli-keyvault/setup.py

```diff
@@ -28,6 +28,7 @@
 
 DEPENDENCIES = [
     'azure-mgmt-keyvault==0.30.0',
+    'azure-keyvault',
```

Comment:
Should specify version `azure-keyvault==0.1.0`.
We don't want an update to `azure-keyvault` to break things.

This is what we do for `azure-storage` also.

When we have profile support in general, we'll revisit patterns for data-plane SDK versions in general.